metricbeat/module/postgresql/activity/activity.go

```diff
@@ -20,16 +20,19 @@ package activity
 import (
 	"database/sql"
 
+	"github.com/elastic/beats/libbeat/logp"
```

Comment:
Nit: Group beats imports

Reply:
Done. For some reason I thought that this was done by `make fmt` command because it usually reorganizes imports

Reply:
it partially does but if one is organised on it's own, it does not catch it. @andrewvc also likes this one ;-)